plugins/outputs/kinesis/kinesis_test.go

```diff
@@ -253,6 +568,38 @@ func (m *mockKinesisPutRecords) SetupResponse(
 	})
 }
 
+func (m *mockKinesisPutRecords) SetupGenericResponse(
+	successfulRecordCount int64,
+	failedRecordCount int64,
+) {
+
+	errorCode := "InternalFailure"
+	errorMessage := "Internal Service Failure"
+	shard := "shardId-000000000003"
+
+	records := []*kinesis.PutRecordsResultEntry{}
+
+	for i := int64(0); i < successfulRecordCount; i++ {
+
```

Comment:
Can you please reduce the usage of newlines to occasions where it helps structuring the code visually (e.g. two lines down).

Reply:
Lol.  I find the newlines helpful, but I'm visually impaired, so I I know I'm bias.  :)  Let me know if that's better for you.